### ISSUE TITLE
Cask Applications dir symlinks not generating?

### ISSUE THREAD

**Issue by jacobgraf** [holman/dotfiles#235]
I ran script/bootstrap and then script/install and it appears that cask has successfully installed the apps I listed, but only a few of them symlinks into the "Applications" directory. There are none in the ~/Applications directory either. Any ideas what would cause some/most of them to install, but not symlink? I even ran 'dot' a few times but the symlinks don't generate then either. Thanks for your help!

**Comment by holman** [comment]
You know, I'm not sure! I looked into things briefly and nothing really jumped out at me. Our usage of Homebrew isn't really too specialized here, so I kind of wonder if it's something with Homebrew that's worth looking into. Maybe there's some additional debug you can print out if you `brew bundle` manually? Or maybe there's some permissions mis-set in your `~/Applications` dir?

**Comment by jacobgraf** [comment]
Thanks for the quick response. I just did a fresh install of homebrew and that didn't fix it. Also, in the Brewfile script, you have the appdir set to /Applications, not ~/Applications. I definitely would prefer them to go into /Applications. Hmmm. Also, it's weird that some of them would get properly symlinked and others don't. There isn't a 'verbose' mode or anything I can enable to get more details, is there?

**Comment by holman** [comment]
> There isn't a 'verbose' mode or anything I can enable to get more details, is there?

Yeah, try running `brew bundle -v`.

**Comment by jacobgraf** [comment]
Ahh. I think I figured it out. I think it has to do with the fact that when I first ran the scripts, I had a few of the apps already installed on my Mac. Just for consistencies sake, I deleted the apps and re-ran the dot command hoping they would fall into line. The problem was that the app was already flagged as 'installed' by cask. I ran brew cask uninstall appname and then re-ran the dot script and everything worked as expected! Thanks again!
